src/scenes/Organizations/Detail/EditOrganization/EditOrganization.tsx

```diff
@@ -0,0 +1,60 @@
+import { useSnackbar } from 'notistack';
+import React, { FC } from 'react';
+import { Scalars, UpdateOrganizationInput } from '../../../../api';
+import { DialogState } from '../../../../components/Dialog';
+import { DialogForm } from '../../../../components/Dialog/DialogForm';
+import { SubmitError, TextField } from '../../../../components/form';
+import { OrganizationDocument } from '../OrganizationDetail.generated';
+import { useUpdateOrganizationMutation } from './EditOrganization.generated';
+
+type EditOrganizationFormProps = DialogState & { orgId: Scalars['String'] };
+
+export const EditOrganizationForm: FC<EditOrganizationFormProps> = (props) => {
+  const { orgId, ...rest } = props;
+  const [updateOrg] = useUpdateOrganizationMutation();
+  const { enqueueSnackbar } = useSnackbar();
+  const submit = async (input: UpdateOrganizationInput) => {
+    const organizationInputWithId = {
+      organization: {
+        ...input.organization,
+        id: orgId,
+      },
+    };
+    const res = await updateOrg({
+      variables: { input: organizationInputWithId },
+      refetchQueries: [
+        {
+          query: OrganizationDocument,
+          variables: { input: orgId },
+        },
+      ],
+    });
+
+    enqueueSnackbar(
+      `Organization Name updated to ${res.data?.updateOrganization.organization.name.value}`,
+      {
+        variant: 'success',
+      }
+    );
+  };
```

Comment:
Let's not use snackbars when changes can be directly seen by user.

```suggestion
```

Reply:
Removed